src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -159,6 +160,40 @@ public async Task<Either<ActionResult, Unit>> DeleteTheme(Guid themeId)
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> DeleteUITestThemes(CancellationToken cancellationToken = default)
+        {
+            return await _userService.CheckCanManageAllTaxonomy()
+                .OnSuccess(_ => _context.Themes
+                    .Where(theme => theme.Title.StartsWith("UI test theme") || theme.Title.StartsWith("Seed theme"))
```

Comment:
I think we allowed deleting of a Seed theme in case they get messed up or something. But we wouldn't necessarily want to be forced to remove them alongside the UI test themes. 

So in the context of removing a specific theme we want to be able to remove any theme related to the UI tests or Seed data. But I think when cleaning up the UI test themes specifically, we won't want to remove the seed themes - just all the UI test themes.